src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,72 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    @JoinTable(name = "movie_sessions_movies",
+            joinColumns = @JoinColumn(name = "movie_session_id"),
+            inverseJoinColumns = @JoinColumn(name = "movie_id"))
+    private Movie movie;
+    @ManyToOne
+    @JoinTable(name = "movie_sessions_cinema_halls",
+            joinColumns = @JoinColumn(name = "movie_session_id"),
+            inverseJoinColumns = @JoinColumn(name = "cinema_hall_id"))
```

Comment:
As with the `movie` field, `@JoinTable` should be replaced with `@JoinColumn` for the `cinemaHall` field. This will correctly map the many-to-one relationship between `MovieSession` and `CinemaHall`.